specification/multicast-notification-service.yaml

```
@@ -286,35 +207,88 @@ paths:
                   properties:
                     name:
                       type: "string"
-                      example: "The Organisation"
                       description: "Organisation name"
                     identifier:
                       type: "object"
                       required:
                         - "system"
                         - "value"
-                      description: "Must follow this identifier system: https://fhir.nhs.uk/Id/ods-organization-code and provide the ODS Code as the value."
+                      description: "Must follow this identifier system: https://fhir.nhs.uk/Id/nhsSpineASID and provide the ASID Code as the value."
                       properties:
                         system:
                           type: "string"
-                          example: "https://fhir.nhs.uk/{Id}/ods-organization-code"
                         value:
                           type: "string"
-                          example: "ABC123"
                 time:
                   type: "string"
                   format: "date-time"
-                  example: "2022-04-05T17:31:00Z"
                   description: "ISO-8601 datetime."
-                dataContentType:
-                  type: "string"
-                  example: "application/fhir+json"
-                  description: "This field describes the content type of the original event payload, if it's available e.g. text/xml, application/json, application/xml+FHIR."
                 data:
-                  type: "string"
-                  example: "https://test-api.nhs.uk/personal-demographics/FHIR/R4/Patient/9912003888"
-                  description: "String containing a pointer to the location of the information, where there exists a retrieval mechanism. E.g. an event generated from PDS would result in a pointer to the patient's PDS Record. The decision is for the events to be purely a notification and not contain any payload. A subscribing system would have to query the source of truth (in our example, PDS) to find what has changed."
-              
+                  type: "object"
+                  properties: 
+                    versionId: 
+                      type: "string"
+                      description: "The versionId/ETAG identifies the version of the resource."
+                    fullUrl: 
+                      type: "string"
+                      description: "The URL where the subscriber can retrieve the source of truth record for the patient or the full event payload."
+                    provenance: 
+                      type: "object"
+                      description: "The provenance describes which actor was responsible for the creation of the event."
```

```suggestion
                      description: "The provenance describes which organisation was responsible for the creation of the event."
```
Sorry, my wording was bad. I'll update on the Confluence page.